modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -294,20 +306,20 @@ Creates a permission policy for a specified entity.
 |Name
 |Description
 |Type
-|Requirement
+|Presence
 
 |`entityReference`
-|Reference values of an entity including namespace and name
+|Reference values of an entity including `kind`, `namespace`, and `name`
 |String
 |Required
 
 |`permission`
-|Type of the permission
+|Permission from a specific plugin, resource type, or name
```

Comment:
Do we need to format `resource type` or `name` in backticks because we use them as values in a request?

please make this change globally, if applicable.

Reply:
Those are types of permissions that we support. An example of a resource type permission would be `catalog-entity` and an example of a named permission would be `catalog.entity.create`.